### PR TITLE
Add dotnet logo to contexts runtime

### PR DESCRIPTION
**Pull request by bruno-garcia**
Resolves ##23465

### CODE REVIEW

**Comment by bruno-garcia** [comment]
That's an angry CI

File: src/sentry/static/sentry/less/group-detail.less [hunk 1]
```diff
@@ -227,6 +227,10 @@
       background-image: url(../images/logos/context/netcore.svg);
     }
 
+    &.net .context-item-icon {
+      background-image: url(../images/logos/context/dotnet.svg);
```
Comment (bruno-garcia) [inline src/sentry/static/sentry/less/group-detail.less:231]: Oh, this isn't from platformicons?

Comment (bruno-garcia) [reply src/sentry/static/sentry/less/group-detail.less:231]: I see: https://github.com/getsentry/sentry/tree/master/src/sentry/static/sentry/images/logos/context

Should I copy the file here? 
I want to add this: https://github.com/getsentry/platformicons/blob/master/svg/dotnet.svg

Comment (matejminar) [reply src/sentry/static/sentry/less/group-detail.less:231]: That's right - I pushed it there.
It's gonna look like this:

image

If you prefer it to be rounded like this:

image
You might wanna add:
```suggestion
      background-image: url(../images/logos/context/dotnet.svg);
      border-radius: 50%;
```

Comment (matejminar) [reply src/sentry/static/sentry/less/group-detail.less:231]: In the future we might wanna refactor these context logos to use the ones directly in the platformicons repo - some of them are now duplicated in both places.

**Review by billyvg — APPROVED** [review]

**Review by untitaker — APPROVED** [review]

**Review by wedamija — APPROVED** [review]

**Comment by markstory** [comment]
:wave: Hi Bruno

**Comment by mitsuhiko** [comment]
👋

**Comment by billyvg** [comment]
👋

**Comment by dcramer** [comment]
i think you forgot to add tests

**Pull request closed without merging**